test/s3path.jl

```diff
@@ -263,5 +263,22 @@ end
         verify_files(temp_path)
 
         rm(temp_path, force=true, recursive=true)
-     end
+    end
+
```

Comment:
Could you just add in a couple tests w/ `*`? And then I think it's good to go, we don't seem to have any for that functionality in here.

Reply:
added